components.json

```diff
@@ -0,0 +1,56 @@
+{
+  "mailjet-send-email": {
+    "componentId": "mailjet-send-email",
+    "title": "Send Email",
+    "imgSrc": "src/assets/images/icons/send-email.svg",
+    "type": "sendEmailComponent",
+    "metadata": {
+      "icon-color": "#7ab260"
+    },
+    "ports": [
+      {
+        "id": "sent",
+        "title": "Sent",
+        "description": "Denotes the state when email is sent"
+      },
+      {
+        "id": "bounced",
+        "title": "Bounced",
+        "description": "Denotes the state when email bounces"
+      },
+      {
+        "id": "error",
+        "title": "Error",
+        "description": "Denotes the state when anyother error occurs"
+      }
+    ],
+    "properties": [
+      {
```

Comment:
Wrap this into a properties category called Basic. We also need to add a new category to have API Key as one of the property, in options of that property say "propertyType:"global". So, we have the same API key in all the mailjet components.